executor/splittest/split_table_test.go

```diff
@@ -266,26 +267,37 @@ func TestShowTableRegion(t *testing.T) {
 	// 4 regions to store record data.
 	// 1 region to store index data.
 	require.Len(t, rows, 5)
-	require.Len(t, rows[0], 11)
+	require.Len(t, rows[0], 13)
 	tbl := external.GetTableByName(t, tk, "test", "t_regions")
 	// Check the region start key.
 	require.Equal(t, fmt.Sprintf("t_%d_r", tbl.Meta().ID), rows[0][1])
 	require.Equal(t, fmt.Sprintf("t_%d_r_-5000", tbl.Meta().ID), rows[1][1])
 	require.Equal(t, fmt.Sprintf("t_%d_r_0", tbl.Meta().ID), rows[2][1])
 	require.Equal(t, fmt.Sprintf("t_%d_r_5000", tbl.Meta().ID), rows[3][1])
 	require.Equal(t, fmt.Sprintf("t_%d_r", tbl.Meta().ID), rows[4][2])
+	// Check scheduling constraint and scheduling state default value
+	for i := range rows {
+		require.Equal(t, "", rows[i][11])
+		require.Equal(t, new(infosync.PlacementScheduleState).String(), rows[i][12])
```

Comment:
Why not explicitly `infosync.PlacementScheduleState.PENDING.String()`? Or maybe just "PENDING"?

Reply:
Hmm `infosync.PlacementScheduleState.PENDING.String() ` doesn't work and I think to reference the property instead of `PENDING` string is a better practice. So when someone update the property value, it won't break.

Reply:
`require.Equal(t, infosync.PlacementScheduleStatePending.String(), rows[0][12])`

This does work.